apps/client/src/views/admin/ListUser.vue

```diff
@@ -0,0 +1,47 @@
+<script setup>
+import { onMounted, ref } from 'vue'
+import { useAdminUserStore } from '@/stores/admin/user.js'
+
+const adminUserStore = useAdminUserStore()
+
+const allUsers = ref([])
+
+const title = 'Liste des utilisateurs'
+const headers = [
+  'Prénom',
+  'Nom',
+  'Courriel',
+  'Création',
+  'Modification',
+]
+const rows = ref([])
+
+const getAllUsers = async () => {
+  allUsers.value = await adminUserStore.getAllUsers()
+}
+
+onMounted(async () => {
+  await getAllUsers()
+  rows.value = allUsers.value?.map(element => ({
+    firstName: element.firstName,
+    lastName: element.lastName,
+    email: element.email,
+    createdAt: formatDate(element.createdAt),
+    updatedAt: formatDate(element.updatedAt),
+  }))
```

Comment:
Ici on prend un warning `Invalid prop: type check failed for prop "rowData". Expected Array, got Object` parce que chacune de nos `rows` sont ici des objets (`rows`  étant un tableau d'objets), il faudrait ajouter un `Object.values(<notre_objet>)` pour convertir l'objet en tableau